core/installation/steps/upgrade_perform.php

```diff
@@ -1,46 +0,0 @@
-<?php
-
-$scripts = [
-    '
-    <script>
-        $(document).ready(function() {
-            $.post("?step=ajax_initialise&initialise=upgrade", {perform: "true"}, function(response) {
```

Comment:
In the `ajax_initialise` file, there are more references and code that does the actual conversion - that should all be torn out too